docs/IS_THIS_A_RAT.md

```diff
@@ -0,0 +1,25 @@
+There are websites that will claim SkyHanni is a rat.
+
+Those websites are wrong.
+
+Those websites might claim to know the mod has malicious code.
+
+Looking closer at the parts in the code they claim are malicious, we notice that their detection algorhytmus has a lot of flaws:
+
+for example:
+
+example 1
+
+Those websites might even claim their findings are powered by artifical intelligence. this is obviously wrong.
+
+Please dont trust websites that claim to know wether or not any mod can rat you or harm your computer.
+
+To download a mod safely, search for `github <mod name>` on the web.
```

Comment:
This is not strictly true. Especially NEU gets deranked, since forks aren't displayed as prominently. I would recommend instead linking to a trusted community hub or referring to the hypixel forums, since those are usually well moderated and link the official github.

Reply:
The hypixel forum post that comes up first is https://hypixel.net/threads/notenoughupdates-neu-skyblock-mod-release.3135465/.

There the same links are provided as the first 3-4 google search results, at least for me.

Reply:
I still agree with nea here as in the past we have seen fake repositories take over real ones and while this may not happen for some mods like skyhanni or neu right now there is no guarantee that it doesnt happen in the future or to another legit mod